app/javascript/stylesheets/blocks/_base.scss

```diff
@@ -8,8 +8,9 @@ h1, h2, h3, h4, h5, h6 {
 
 .button.new-right {
   float: right;
-  margin-top: 20px;
   margin-left: 20px;
+  position: relative;
+  z-index: 100;
```

Comment:
That's because of the datatable upgrade. Those buttons are included in the datatable div which updated its CSS.